app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -96,6 +96,26 @@ def save_in_person_enrollment(user, profile)
         enrollment = InPersonEnrollment.create!(
           profile: profile,
           user: user,
+          selected_location_details: {
+            'name' => 'BALTIMORE — Post Office™',
+            'streetAddress' => '900 E FAYETTE ST RM 118',
+            'city' => 'BALTIMORE',
+            'state' => 'MD',
+            'zip5' => '21233',
+            'zip4' => '9715',
+            'phone' => '555-123-6409',
+            'hours' => [
+              {
+                'weekdayHours' => '8:30 AM - 4:30 PM',
+              },
+              {
+                'saturdayHours' => '9:00 AM - 12:00 PM',
+              },
+              {
+                'sundayHours' => 'Closed',
+              },
+            ],
+          },
```

Comment:
This is a temporary solution. Once the user's location selection is saved to the session we will begin using that value instead of this hard-coded value